contracts/curve/AnteCurveLusdPoolCompositionTest.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+import {IERC20Metadata} from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+/// @title Curve lusd Pool Composition Test
+/// @notice Checks that the LUSD balance, and the DAI, USDC, USDT balances in Curve 3Pool scaled by 
+///         the amount of 3CRV in Curve lusd, are each < 90% of Curve tusd pool's total token balance.
+contract AnteCurveLusdPoolCompositionTest is AnteTest("LUSD, DAI, USDC, USDT balances are each < 90% of Curve lusd pool balance") {
+    address public constant CURVE_LUSD_POOL_ADDRESS = 0xEd279fDD11cA84bEef15AF5D39BB4d4bEE23F0cA;
+    address public constant CURVE_3POOL_POOL_ADDRESS = 0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7;
+    address public constant CURVE_3CRV_ADDRESS = 0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490;
+    address public constant LUSD_ADDRESS = 0x5f98805A4E8be255a32880FDeC7F6728C6568bA0;
+    address public constant DAI_ADDRESS = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public constant USDC_ADDRESS = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address public constant USDT_ADDRESS = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+
+    IERC20Metadata private constant LUSD = IERC20Metadata(LUSD_ADDRESS);
+    IERC20Metadata private constant CURVE_3CRV = IERC20Metadata(CURVE_3CRV_ADDRESS);
+    IERC20Metadata private constant DAI = IERC20Metadata(DAI_ADDRESS);
+    IERC20Metadata private constant USDC = IERC20Metadata(USDC_ADDRESS);
+    IERC20Metadata private constant USDT = IERC20Metadata(USDT_ADDRESS);
+
+    constructor() {
+        testedContracts = [CURVE_LUSD_POOL_ADDRESS];
+        protocolName = "Curve";
+    }
+
+    /// @return true if the balances of each token are < 90%
+    function checkTestPasses() public view override returns (bool) {
+        uint256 LUSDBalance = LUSD.balanceOf(CURVE_LUSD_POOL_ADDRESS) / 10**LUSD.decimals();
+        uint256 DAIBalance = DAI.balanceOf(CURVE_3POOL_POOL_ADDRESS) / 10**DAI.decimals();
+        uint256 USDCBalance = USDC.balanceOf(CURVE_3POOL_POOL_ADDRESS) / 10**USDC.decimals();
+        uint256 USDTBalance = USDT.balanceOf(CURVE_3POOL_POOL_ADDRESS) / 10**USDT.decimals();
```

Comment:
Let's add a little clarification either by a comment above saying that this is from the 3Pool or update the variable name